src/pep257.py

```diff
@@ -128,6 +128,17 @@ def __iter__(self):
     def _publicity(self):
         return {True: 'public', False: 'private'}[self.is_public]
 
+    @property
+    def source(self):
+        """Return the source code for the definition."""
+        full_src = self._source[self._slice]
+
+        def predicate(line):
```

Comment:
A better name might be nice, e.g., `is_comment_or_empty`. `predicate` is a bit too generic.